lib/ngx/errlog.md

```diff
@@ -13,6 +13,7 @@ Table of Contents
 * [Methods](#methods)
     * [set_filter_level](#set_filter_level)
     * [get_logs](#get_logs)
+    * [get_sys_filtering_level](#get_sys_filtering_level)
```

Comment:
I think we should use `get_sys_filter_level` instead of `get_sys_filtering_level` to be more consistent with the existing `set_filter_level` API function.